### PR TITLE
[script.myepisodecalendar] v1.4.1

### PR DESCRIPTION
**Pull request by Connum**
### Description
+ Fixed login by switching myepisodecalendar URLs to https
+ Switched TVDB API URLs to https
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.myepisodecalendar/addon.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.myepisodecalendar"
        name="MyEpisodeCalendar"
-       version="1.4.0"
+       version="1.4.1"
        provider-name="Connum">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
```
Comment (Rechi) [inline script.myepisodecalendar/addon.xml]: > WARN: For jarvis it is advised to set xbmc.python version to 2.24.0

Comment (Rechi) [reply script.myepisodecalendar/addon.xml]: This is only a warning, it's not a mandatory to change it.

Comment (Connum) [reply script.myepisodecalendar/addon.xml]: Alright, thanks for the info, let me know if there's anyhing you need me to do!

**Comment by Connum** [comment]
Thanks! I'll do so with the next update, already changed it in my code base but would like the login fix to be available through the repo as soon as possible!

**Comment by Rechi** [comment]
As said you can ignore the warnings, but the errors have to be fixed.

**Comment by Connum** [comment]
Ok, so I found out that one error is a result from changes made to the Travis schema validation regex (see #168) and was able to locate and remove the transparency in the icon.png file. Where do I find any information on the old/new language directory structure? I haven't been able to get any information about that from the wiki or a Google search...

**Comment by Connum** [comment]
Ok, found it... https://forum.kodi.tv/showthread.php?tid=332789
I'm going to close this PR and make a new one for version 1.4.2, as I already added that as a release on my repo.

**Pull request closed without merging**